setup.sh

```diff
@@ -1,8 +1,10 @@
 #!/bin/sh
 
-
-#echo "$@"
+(
+cd $(dirname ${BASH_SOURCE[0]})
```

Comment:
Needs fixing:

[usarica@lxplus013 MELA]$ cd $(dirname ${BASH_SOURCE[0]})
dirname: missing operand
Try `dirname --help' for more information.